core/App/ehsm_provider.cpp

```diff
@@ -304,6 +304,17 @@ ehsm_status_t Initialize()
         sgx_destroy_enclave(g_enclave_id);
     }
```

Comment:
commit title:
"
Add self-test for all the corresponding algorithms

the self-test vectors will support for:
aes: gcm encryption/decryption
rsa: verify, encryption/decryption
ecc: verify
sm2: verify, encryption/decryption
sm4: cbc, ctr encryption/decryption

test: passed all the unit test
Signed-off-by: WangHaihengX  <haihengx.wang@intel.com>
”